hosts/nix-nextcloud/configuration.nix

```
@@ -107,6 +115,34 @@
         "traefik.http.routers.whiteboard.middlewares" = "strip-whiteboard";
       };
     };
+    "appapi-harp" = {
+      image = "ghcr.io/nextcloud/nextcloud-appapi-harp:release";
+      environment = {
+        "HP_SHARED_KEY" = "some_very_secure_password";
+        "NC_INSTANCE_URL" = "https://nextcloud.sotga.de";
+      };
```

Hier wollen wir wieder eine Environment File haben wegen den Secrets.
Das Secret muss oben noch hinzugefügt werden.

```suggestion
environmentFiles = [ config.sops.secrets.nextcloud_app_api.path ];
```